pkgs/development/python-modules/sip/default.nix

```diff
@@ -15,19 +16,20 @@
 
 buildPythonPackage rec {
   pname = "sip";
-  version = "6.8.3";
+  version = "6.8.6";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-iIVHsBi7JMNq3tUZ6T0+UT1MaqC6VbfMGv+9Rc8Qdiw=";
+    hash = "sha256-f8lZ5I5uxdWvi9Am9p9eJNCLPLirs0IXb1q4AwzAfXo=";
   };
 
   nativeBuildInputs = [ setuptools ];
 
   propagatedBuildInputs = [
```

Comment:
```suggestion
  dependencies = [
```

Reply:
Haven't read up on this changes. Could you ref an overview/doc on these changes for *inputs to dependencies/build-system?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

Reply:
Thanks, aligning with python terms on python packages seems like a great move